A37-xds-aggregate-and-logical-dns-clusters.md

```diff
@@ -95,7 +100,9 @@ For example:
 Each underlying cluster will have an associated discovery mechanism.
 The ordered list of discovery mechanisms will be passed down in the
 config for the new `xds_cluster_resolver_experimental` policy, which is the
-child of the CDS policy.
+child of the CDS policy. If an underlying cluster appears more than once in
+the list of clusters, only one discovery mechanism entry should be created for
+it.
```

Comment:
Maybe add something like ", in the earliest position in the list in which it is seen"?

Reply:
Added